Pod/Classes/NSURLSession/NSURLSession+Mock.swift

```diff
@@ -134,14 +141,24 @@ extension NSURLSession {
     public class func removeAllMocks(of request: NSURLRequest) {
         self.register.removeAllMocks(of: request)
     }
+    
+    /**
+     For a given handle, has it been consumed or not?
+     
+     - paramater handle: The returned handle from a call to mock a request i.e. mockSingle(...)
+     */
+    public class func hasMockConsumed(handle: Handle) -> Bool {
```

Comment:
Not sure if it's possible also, but I guess a mock that was somehow never added to the register will also be reported as consumed